### ISSUE TITLE
Add test cases

### ISSUE THREAD

**Issue by lukasrieger-dev** [bubeck/airspace_germany#2]
In order to ensure that code changes (refactoring) don't change functionality, add test cases for regression testing.

**Comment by bubeck** [comment]
I agree. However, I would like to finish/polish the code first. Probably a week or so, then I am happy to accept any PR doing this. :)
